includes/class-wcs-post-meta-cache-manager.php

```diff
@@ -8,7 +8,6 @@
  *
```

Comment:
Thoughts on attaching a comment here to mention that this class is only to be used for WP Posts? Reference to where this was brought up: https://github.com/Automattic/woocommerce-subscriptions-core/pull/318#discussion_r1035405101